rust/noosphere-ns/src/dht/swarm.rs

```diff
@@ -52,6 +52,7 @@ impl DHTBehaviour {
         let kad = {
             let mut cfg = KademliaConfig::default();
             cfg.set_query_timeout(Duration::from_secs(config.query_timeout.into()));
+            cfg.set_record_filtering(KademliaStoreInserts::FilterBoth);
```

Comment:
Rather than automatically storing propagated records from other nodes, require it to be manually added -- this allows us a hook to perform validation and lookups (not yet implemented, everything is added anyway lol)

Reply:
DHT Store also has its own expiration, which is not yet tied into the NS propagation timer